content/vault/v1.19.x/content/docs/upgrade/index.mdx

```diff
@@ -88,12 +88,13 @@ To perform an in-place upgrade of a single Vault instance:
    
 <Warning title="Isolate your test instance">
 
-If you use Vault to coordinate secrets with third party resources (cloud
-credentials, database credentials, etc.) make sure you isolate the test
-instance from external network connectivity during testing. Isolating the test
-instance prevents it from trying to revoke 3rd party resources. Otherwise, the
-third party resource credentials may expire and prevent access from the
-production cluster.
+Upgrade or restore tests should always be performed in a fully isolated
+network environment. This isolation is critical to prevent both unwanted
+cluster-to-cluster communication (which maintains data consistency) and to
+stop the test instance from attempting to revoke 3rd-party
+credentials (secrets, etc.). If the test instance revokes live credentials,
+they may expire, which might result in irrevocable leases for the production
+Vault cluster from which the snapshot was taken.
```

Comment:
I suggest we make this a partial (per the editing suggestion in PR #1277 since we're including it across multiple docs